resources/testharness.js

```diff
@@ -770,7 +770,7 @@ policies and contribution forms [3].
     }
 
     function done() {
-        if (tests.tests.length === 0) {
+        if (tests.tests.length === 0 && tests.status.status === tests.status.OK) {
```

Comment:
@jgraham do you recall why you made this change? `test.status.status` is initially null, so this changed the effects of very simple single-page tests like resources/tests/functional/single-page-test-no-assertions.html and content-security-policy/script-src/scripthash-default-src.sub.html.

Reply:
I don't remember, it sounds like it should also allow that the status is null.